src/Components/Home/NewQuestionModal.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Modal, Form, Button } from "react-bootstrap";
+
+const NewQuestionModal = (props) => {
+  return (
+    <Modal show={props.show} onHide={props.onHide}>
+      <Modal.Header>
+        <Modal.Title>New Question</Modal.Title>
+      </Modal.Header>
+
+      <Modal.Body>
+        <Form>
+          <Form.Group controlId="formQuestion">
+            <Form.Label>{props.question}</Form.Label>
+            <Form.Control
+              type="text"
+              placeholder="Enter your answer..."
+              value={props.answer}
+              onChange={e => props.onAnswerChange(e.target.value)}
+            />
+          </Form.Group>
+        </Form>
+      </Modal.Body>
+
+      <Modal.Footer>
+        <Button
+          variant="primary"
+          onClick={() =>
+            props.onSaveClick({
+              productId: props.productId,
+              question: props.question,
+              answer: props.answer,
+            })
+          }
+        >
+          Save and close
```

Comment:
what if we have two buttons: one would add to likes, others not? If user has to agree "yeah, I don't have space" - why would s/he get a point to likes? this change is going to mean more development work but feels more logical, imho. Now a user can get a like incremented even if the field is left empty.

what about "save and close" be smt like "Still want to buy" and another one be ... "should reconsider buying"?

Reply:
I'll move this to a backlog card, I feel like this needs discussion